src/nvim/eval.c

```diff
@@ -1186,7 +1186,7 @@ int call_vim_function(
     const char_u *func,
     int argc,
     const char_u *const *const argv,
-    int safe,                       // use the sandbox
+    _Bool safe,                       // use the sandbox
```

Comment:
use `bool`

Reply:
@hyatskov this `_Bool` still needs to be changed.

 Also the PR has conflicts, needs a rebase.

Reply:
`_Bool` needs to be changed to `bool`.

Reply:
My bad.